region/norfair/east/Magdollite Tunnel.json

```diff
@@ -88,8 +88,38 @@
       "name": "Magdollite Farm",
       "requires": [
         "h_heatProof",
-        {"refill": ["Energy", "Missile"]}
+        {"cycleFrames": 310}
       ],
+      "farmCycleDrops": [{"enemy": "Magdollite", "count": 3}],
+      "flashSuitChecked": true
+    },
+    {
+      "name": "Multiviola Farm",
```

Comment:
I'm not sure if there's a case where the Multiviola farm would have a benefit over the Magdollite farm? It looks like Multiviolas just have worse drops while also being slower? I guess it doesn't hurt to leave it in,  to support the possibility of drop stats being altered.

Reply:
So I checked if multiviolas dropped supers and PBs but didn't do the same for magdolites for some reason.
I'll remove the multiviola farms.